src/js/view/controlbar.js

```diff
@@ -90,7 +126,7 @@ define([
                 mute: muteButton,
                 volume: volumeSlider,
                 volumetooltip: volumeTooltip,
-                cast: button('jw-icon-cast jw-off', this._api.castToggle, this._localization.cast),
+                cast: !utils.isSafari()? castButton(this._localization.cast) : button('jw-icon-airplay jw-off', this._api.castToggle, this._localization.airplay),
```

Comment:
I'm not sure `isSafari` is the right check here. There are a few browsers that shouldn't have any cast element. What is your take on this?